src/js/libcs/List.js

```diff
@@ -209,9 +209,9 @@ List.consecutive = function(a) {
 };
 
 List.reverse = function(a) {
-    var erg = [];
-    for (var i = a.value.length - 1; i >= 0; i--) {
-        erg.push(a.value[i]);
+    var erg = new Array(a.value.length);
```

Comment:
Like `var erg = a.value.slice().reverse()`.

@strobelm Maybe you can compare the performance of that?